packages/server/graphql/public/mutations/upsertTeamPromptResponse.ts

```diff
@@ -0,0 +1,81 @@
+import {JSONContent} from '@tiptap/core'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {upsertTeamPromptResponse as upsertTeamPromptResponseQuery} from '../../../postgres/queries/upsertTeamPromptResponses'
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import extractTextFromTipTapJSONContent from '../../mutations/helpers/tiptap/extractTextFromTipTapJSONContent'
+import {MutationResolvers} from '../resolverTypes'
+
+const upsertTeamPromptResponse: MutationResolvers['upsertTeamPromptResponse'] = async (
+  _source,
+  {teamPromptResponse},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const viewerId = getUserId(authToken)
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+  const {teamPromptResponseId, meetingId, content} = teamPromptResponse
+
+  // AUTH
+  const promptResponse = teamPromptResponseId
+    ? await dataLoader.get('teamPromptResponses').loadNonNull(teamPromptResponseId)
+    : {
+        meetingId,
+        userId: viewerId,
+        sortOrder: 0,
+        content: {},
+        plaintextContent: ''
+      }
+  if (!promptResponse) {
+    return standardError(new Error('PromptResponse not found'), {userId: viewerId})
+  }
```

Comment:
-1 this will never happen, now if there's no response in db we provide manually created object